js/map.js

```diff
@@ -29,7 +29,7 @@ $(document).ready(function() {
 
     var map = L.map('map').setView([51.0475378, 3.7261835], 13);
     var osm = L.tileLayer('https://{s}.tiles.mapbox.com/v3/feliciaan.keoaj8d5/{z}/{x}/{y}.png', {
-      attribution: '<a href="http://www.mapbox.com/about/maps/" target="_blank">Mapbox</a> | Made with ❤ by <a href="http://zeus.ugent.be">Zeus WPI</a>'
+      attribution: '© <a href="http://www.mapbox.com/about/maps/" target="_blank">Mapbox</a> | © <a href="http://www.openstreetmap.org/copyright">OpenStreetMap contributors</a> | Made with ❤ by <a href="http://zeus.ugent.be">Zeus WPI</a>'
```

Comment:
Put a blank target on the link. 
Refer to the HTTPS site, also for mapbox and zeus